src/main/java/util/Validator.java

```diff
@@ -0,0 +1,26 @@
+package util;
+
+import model.exceptions.NoContactDataException;
+
+import java.util.regex.Pattern;
+
+public class Validator {
+
+    private static String emailTemplate = "^[a-zA-Z0-9._%+-]++@[a-zA-Z0-9-]{2,}\\.[a-zA-Z]{2,}+$";
+    private static String phoneTemplate = "^\\+375(29|33|44|25|17)\\d{7}$";
+
+    public static Boolean isEmailValid(String email) {
```

Comment:
you can return primitive boolean instead of Boolean as object type. I will save space and time skipping convertation to primitive and back